styles/about.css

```diff
@@ -75,8 +75,20 @@
     align-items: center;
     flex-direction: column;
     max-width: 45%;
+    animation-name: about-us-col-img;
+    animation-duration: 5s;
+    animation-iteration-count: infinite;
   }
-  
```

Comment:
Use margin-bottom instead of margin-top and reduce the percentage from 10% to 5%.